<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShotPage</class>
 <widget class="QWidget" name="ShotPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="ShootScreenTitle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 22pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Shooting Screen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="ShootingFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ShootingGrid">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="shipsRemainLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
text-decoration: underline;
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Sunk Ships</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="shootScreenEndTurn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 18pt &quot;MS Shell Dlg 2&quot;;
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>End Turn</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QLabel" name="customSunkLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Custom Ships Sunk: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="bikesSunkLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Bikes Sunk: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="bikeSunkCount">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="carSunkCount">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="busSunkCount">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="customSunkCount">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="carsSunkLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Cars Sunk:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="busesSunkLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Buses Sunk:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="1">
     <widget class="QLabel" name="totalHitsCount">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="totalHitsLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Total Hits</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="totalShotsCount">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="accuracyCount">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="accuracyLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Shot Accuracy</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="totalShotsLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Total Shots</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="shipsRemainLabel_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
text-decoration: underline;
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Player Stats</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="indicatorFrame">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#indicatorFrame{
	border: 5px solid white;
	border-radius: 8px;
};</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="indiTitleLabelShotPage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Turn Indicator:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="turnIndiShotPage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>P2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
